packages/pangea-sdk/pangea/services/authn/authn.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2022 Pangea Cyber Corporation
+# Author: Pangea Cyber Corporation
+
+from typing import Optional
+
+from pangea.response import PangeaResponse
+from pangea.services.authn.models import (
+    IDProvider,
+    PasswordUpdateRequest,
+    PasswordUpdateResult,
+    Profile,
+    Scopes,
+    UserCreateRequest,
+    UserCreateResult,
+    UserDeleteRequest,
+    UserDeleteResult,
+    UserInviteDeleteRequest,
+    UserInviteDeleteResult,
+    UserInviteListResult,
+    UserInviteRequest,
+    UserInviteResult,
+    UserListRequest,
+    UserListResult,
+    UserLoginRequest,
+    UserLoginResult,
+    UserProfileGetRequest,
+    UserProfileGetResult,
+    UserProfileUpdateRequest,
+    UserProfileUpdateResult,
+)
+from pangea.services.base import ServiceBase
+
+
+class AuthN(ServiceBase):
+    """AuthN service client.
+
+    Provides methods to interact with the [Pangea AuthN Service](https://pangea.cloud/docs/api/authn).
+
+    The following information is needed:
+        PANGEA_TOKEN - service token which can be found on the Pangea User
+            Console at [https://console.pangea.cloud/project/tokens](https://console.pangea.cloud/project/tokens)
+
+    Examples:
+        import os
+
+        # Pangea SDK
+        from pangea.config import PangeaConfig
+        from pangea.services import AuthN
+
+        PANGEA_TOKEN = os.getenv("PANGEA_AUTHN_TOKEN")
+        authn_config = PangeaConfig(domain="pangea.cloud")
+
+        # Setup Pangea AuthN service
+        authn = AuthN(token=PANGEA_TOKEN, config=authn_config)
+    """
+
+    service_name: str = "authn"
+    version: str = "v1"
+
+    def __init__(
+        self,
+        token,
+        config=None,
+    ):
+        super().__init__(token, config)
+
+    #   - path: authn::/v1/password/update
+    # https://dev.pangea.cloud/docs/api/authn#change-a-users-password
```

Comment:
I know this is internal, are we worried about confusing users who navigate to this url only to get turned back with a 403?

I know the docs are only there for now, maybe we should have a TODO to change this back to pangea.cloud/docs?